### ISSUE TITLE
Games don't start with Proton 5.0

### ISSUE THREAD

**Issue by vhda** [ValveSoftware/Proton#4083]
Most recently I only tested Skyrim SE and Subnautica Below Zero.
Both start with Proton 4.11, but fail to start with 5.0 with various exceptions and segmentation faults showing up in the log:
[Proton Log](https://pastebin.com/B6gGriX8)

[Steam System Information](https://pastebin.com/YiUWWpqY)

**Comment by x4mers** [comment]
As of 5.0, DX to Vulkan translation is enabled by default with D9VK.

Your logs show you're using a Radeon 7900, which doesn't have vulkan support enabled by default. It has to be enabled by telling the kernel to use amdgpu instead of radeon for older AMD cards. If you run inxi -xxxG in a terminal, it likely says you're using the radeon driver instead of amdgpu.

You need to add the following to your kernel command line to enable support for your card.

radeon.si_support=0 amdgpu.si_support=1 radeon.cik_support=0 amdgpu.cik_support=1

**Comment by vhda** [comment]
Hi @x4mers,

Unfortunately, I already have amdgpu enabled, so that can't be the reason. I can share my vulkaninfo output if it helps.

**Comment by JacobSvenningsen** [comment]
Hello, from the log and the steam system information, I'm quite confident you've run into an issue I, and some others have experienced as well https://github.com/ValveSoftware/Proton/issues/3573
You have a very old lcd for the radv vulkan driver lying around, version 1.0.3. When using proton, it apparently picks up this lcd version, and DXVK then fails because it requires at least version 1.1.

Try deleting **/etc/vulkan/icd.d/radeon_icd.i686.json**, **/etc/vulkan/icd.d/radeon_icd.x86_64.json** and **/etc/vulkan/icd.d/radeon_icd.json** then restart steam.
See if those lcd's stop showing up in the Steam System Information window. If they don't, and if you're still using RadV, which it looks like from the Proton Log, then the games should launch

**Comment by vhda** [comment]
Well... that works! Thanks!

But dpkg's behavior was quite odd:

    $ dpkg -S /etc/vulkan/icd.d/radeon_icd\*
    libgl1-mesa-dri:i386, libgl1-mesa-dri:amd64: /etc/vulkan/icd.d/radeon_icd.json
    libgl1-mesa-dri:amd64: /etc/vulkan/icd.d/radeon_icd.x86_64.json
    libgl1-mesa-dri:i386: /etc/vulkan/icd.d/radeon_icd.i686.json

I then force reinstalled libgl1-mesa-dri packages and now:

    $ dpkg -S /etc/vulkan/icd.d/radeon_icd\*
    dpkg-query: no path found matching pattern /etc/vulkan/icd.d/radeon_icd*

Basically, after reinstalling libgl1-mesa-dri when those files were no longer there dpkg no longer recognizes those files! I confirmed this by noticing that intel_icd was still recognized even after reinstalling, so I deleted the icd files, reinstalled packages and, again, dpkg no longer recognized them.
I should report this as a bug to Mesa...
